### PR TITLE
added unit tests for images_sum and images_mean

### PR DESCRIPTION
**Pull request by vivekthampy**
@stuwilkins ((Attn @mpmdean) Added the tests. Have not done this before. Hopefully it is correct.

### CODE REVIEW

File: tests/test_image.py [hunk 1]
```diff
@@ -79,3 +79,22 @@ def test_stackstderr():
...
     assert_array_equal(n, np.ones((20, 20), dtype=np.float32) * 1000.0)
+
+
+def test_images_mean():
+    x = []
+    for ii in range(1,11):
+        x.append(np.repeat(ii*np.ones(ii*100, dtype=np.float32), 400).reshape(
+        (ii*100, 20, 20)))
+    x = np.array(x)
+    m = images_mean(x)
```
Comment (stuwilkins) [inline tests/test_image.py]: This needs an ``assert`` statement

File: tests/test_image.py [hunk 2]
```diff
@@ -79,3 +79,22 @@ def test_stackstderr():
...
+
+
+def test_images_sum():
+    x = []
+    for ii in range(1,11):
+        x.append(np.repeat(ii*np.ones(ii*100, dtype=np.float32), 400).reshape(
```
Comment (stuwilkins) [inline tests/test_image.py]: This is very contrived ... Is there not a cleaner way to do this using ``np.ones``

File: tests/test_image.py [hunk 3]
```diff
@@ -79,3 +79,22 @@ def test_stackstderr():
     m, n = stackstderr(x)
     assert_array_almost_equal(m, np.std(x, axis=0) / np.sqrt(n), 3)
     assert_array_equal(n, np.ones((20, 20), dtype=np.float32) * 1000.0)
+
+
+def test_images_mean():
+    x = []
+    for ii in range(1,11):
+        x.append(np.repeat(ii*np.ones(ii*100, dtype=np.float32), 400).reshape(
```
Comment (stuwilkins) [inline tests/test_image.py]: This is very contrived, is there not a more numpy way of doing this?

**Comment by stuwilkins** [comment]
@vivekthampy the tests are failing, can you please fix this up. In future it is usually a good thing to check the Travis build after opening the PR and fix up then. 

Also, you can always open the PR before you plan to merge - I often do this - and then comment when you are ready to review.

**Comment by vivekthampy** [comment]
Just corrected the bug. Testing the build now. Will test in the future before making a pull request.

As for the test, I was trying to have a general use case where the number of images in each even can be different. Not sure how I would do it in a more numpy way. I'll think about it. Any suggestions are welcome.

**Pull request merged**